assets/js/frontend/add-to-cart.js

```diff
@@ -113,21 +112,6 @@ jQuery( function( $ ) {
 		}
 	};
 
-	/**
-	 * Update cart page elements after add to cart events.
-	 */
-	AddToCartHandler.prototype.updateCartPage = function() {
-		var page = window.location.toString().replace( 'add-to-cart', 'added-to-cart' );
-
-		$.get( page, function( data ) {
-			$( '.shop_table.cart:eq(0)' ).replaceWith( $( data ).find( '.shop_table.cart:eq(0)' ) );
```

Comment:
These are used to update cart totals, table etc in `/cart` page, so removing these will break that page. 
To test, add some products in cart, click on view cart, and then add some thing from `You may be interested in` section, total will not change after applying these changes.

Instead of removing this event handler, perhaps we can detect if `shop_table` exists on current page before making the request for updated page?